sourceCode.py

```diff
@@ -1 +1,523 @@
+## ___________________________________________________________________________________________________________________________________________  ##
+##                                                                                                                                              ##
+##                                          Jordan: a chatbot in times of corona but also in other times                                        ##
+## ____________________________________________________________________________________________________________________________________________ ##
+##                                                                                                                                              ##
+##################################################################################################################################################
+
+                            ########################################################################################  
+                            ##   --------------------------- General Stuff ----------------------------------     ##
+                            ########################################################################################
+
+## hello reviewer, this is by the way the main file to review. Ill be grateful about some tips on how to integrate everything and how to create a web interface thingy
+## I have now created some libraries per topic and functions as you can see, but did not integrate any grammar stuff yet or any more general construct in which the stuff will be embedded in, like choosing a topic. will do that next time. tired now. 
+
+##########################################################
+##                                                      ##
+## grammar lesson 1: first person becomes second person ##
+##                                                      ##
+##########################################################
+
+gReflections = {
+    "am"   : "are",
+    "was"  : "were",
+    "i"    : "you",
+    "i'd"  : "you would",
+    "i've"  : "you have",
+    "i'll"  : "you will",
+    "my"  : "your",
+    "are"  : "am",
+    "you've": "I have",
+    "you'll": "I will",
+    "your"  : "my",
+    "yours"  : "mine",
+    "you"  : "me",
+    "me"  : "you"
+    }
+  
+                                ########################################################################################  
+                                ##   -------------------- Topic Functions and Libraries -------------------------     ##
+                                ########################################################################################
+
+
+
+##############################
+##                          ##
+## small talk with Jordan   ##
+##                          ##
+##############################
+
+## create the library for small-talk with Jordan
+library_smalltalk = [
+    [r'Hi(.*)',
```

Comment:
I think you're using r'...' to avoid the user input being messed up because it accidentally includes one of the string manipulations. That's very clever!